packages/hooks/README.md

```diff
@@ -126,3 +136,21 @@ usePoller(onPoll, {
 | `optional.interval`  | `number`   | What interval the onPoll should be called.                           | `30000` |
 | `optional.immediate` | `boolean`  | If we immediately poll, if false we wait till first interval occurs. | `true`  |
 | `optional.enabled`   | `boolean`  | Is polling enabled.                                                  | `true`  |
+
+## useKeyPress
+
+Hook that accepts a keyCode and returns a boolean based on whether or not that key is being pressed.
+
+### Example
+
+```js
+import { useKeyPress } from '@leafygreen-ui/hooks';
```

Comment:
Can we show an example of using this with lib's `keyMap` variable? Noting it's optional to use it?

Reply:
N/A -- moved useKeyPress to Tabs for now and am not exporting for external consumption